flink-table/flink-sql-gateway/src/test/resources/sql/insert.q

```diff
@@ -0,0 +1,182 @@
+# insert.q - insert
```

Comment:
Not all cases are in SqlClient currently supported?

Reply:
Currently only `ADD JAR/REMOVE JAR/STATEMENT SET(interactive)` is not supported.  I have opened a ticket about the interactive statement set.